UserJourneyRecorder/UserJourneyRecorderWebApp/UserJourneyRecorderWebApp.csproj

```diff
@@ -74,6 +84,7 @@
     <Content Include="packages.config" />
     <Content Include="languages\en-us_102.json" />
     <None Include="Properties\PublishProfiles\B2CRecorder.pubxml" />
+    <None Include="Properties\PublishProfiles\CumminsUserJourney - Web Deploy.pubxml" />
```

Comment:
Is this intended? The file doesn't seem generic enough and isn't included in the PR.

Reply:
Nope. My bad! I'll remove it when I get a chance

Reply:
Removed